app/server/labml_app/analyses/experiments/comparison.py

```diff
@@ -18,6 +18,7 @@ class ComparisonPreferences(preferences.Preferences):
     def defaults(cls):
         return dict(base_series_preferences=[],
                     base_experiment=str,
+                    step_range=(-1, -1),
```

Comment:
change this to a list ?

Reply:
f9fb62656f99de71988cbd251c9d0fc0a5e96954